src/js/db.js

```diff
@@ -183,6 +181,20 @@ console.log('Pass!');
             });
         })());
     }
+    if (upgradeDB.oldVersion <= 18) {
```

Comment:
You also have to change 18 to 19 in `connectLeague` or this will never get triggered.